src/calendar.js

```diff
@@ -214,15 +226,16 @@ angular.module('ui.calendar', [])
   .directive('uiCalendar', ['uiCalendarConfig', function(uiCalendarConfig) {
     return {
       restrict: 'A',
-      scope: {eventSources:'=ngModel',calendarWatchEvent: '&'},
+      scope: {eventSources:'=ngModel',calendarWatchEvent: '&', resources: '='},
       controller: 'uiCalendarCtrl',
       link: function(scope, elm, attrs, controller) {
-
         var sources = scope.eventSources,
+            evResources = scope.resources,
             sourcesChanged = false,
             calendar,
             eventSourcesWatcher = controller.changeWatcher(sources, controller.sourceFingerprint),
             eventsWatcher = controller.changeWatcher(controller.allEvents, controller.eventFingerprint),
+            resourceWatcher = controller.changeWatcher(scope.resources, controller.resourceFingerprint),

```

Comment:
I would change `controller.changeWatcher(scope.resources, controller.resourceFingerprint)` to `controller.changeWatcher(scope.resources || [], controller.resourceFingerprint)`